### ISSUE TITLE
[Request] Summary text in YouTube

### ISSUE THREAD

**Issue by Rock-Anderson** [imelnyk/ArxivPapers#1]
Great work, thanks for this @imelnyk. Really helpful. 

Minor request - 
Can you also please include the summary generated as text in the description section of YouTube for individual videos? 

TIA.

**Comment by imelnyk** [comment]
Currently, I include ~30-word summary of the abstract in the description section. So, to clarify, what else would you like to include there?

**Comment by Rock-Anderson** [comment]
If it's not much of a trouble, I was thinking if it'd be possible to add the full-length summary as well. 
Helps to save as notes while watching these helpful videos, and even to refer to / search through the full summaries later without having to play the entire video. 

Not sure if there'd be any length restrictions to paste directly in the YouTube descriptions. 
Thanks.

**Comment by imelnyk** [comment]
Yeah, I can take a look at this. But at some point I used to add text for subtitles. This text was the GPT-generated content, which I later converted to audio. For example, if you look at this video: https://www.youtube.com/watch?v=JaIL1VAEwZ8 and look at the "Show Transcript" section, the text shown is word-by-word output from GPT. Maybe I can start adding these subtitles again, if they are useful.
